src/App.js

```diff
@@ -2,13 +2,13 @@ import React from 'react';
 import {
   Text, View,
 } from 'react-native';
-import SignInScreen from './SignIn/SignInScreen';
+import MarketplaceScreen from './Marketplace/MarketplaceScreen';
 
 function App() {
   return (
     <View>
-      <Text> APIFM App </Text>
-      <SignInScreen />
+      <Text> APIFM App</Text>
+      <MarketplaceScreen />
```

Comment:
Revert this back to the original (should be showing sign in screen instead of marketplace upon startup). In the future, it's totally ok to change files like these for testing purposes but make sure you change them back before pushing and merging.